src/HdbDevice.h

```diff
@@ -390,7 +413,35 @@ class ArchiveCB : public Tango::CallBack, public Tango::LogAdapter
 	virtual void push_event(Tango::AttrConfEventData* data);
 };
 
+template<typename T>
+void HdbDevice::push_events(const std::string& attr_name, T* data, bool sleep)
+{
+    try
+    {
+        _device->push_change_event(attr_name, data);
+        _device->push_archive_event(attr_name, data);
+    }
+    catch(Tango::DevFailed &e){}
+
+    // TODO is this needed ?
+    if(sleep)
+        usleep(1 * s_to_ms_factor);
```

Comment:
It's the same value, but correct const here is ms_to_us_factor

Reply:
Indeed, well spotted, thanks!